### PR TITLE
#50727 new alias for current_database and added current_schemas

### PR DESCRIPTION
**Pull request by priera**
### Changelog category (leave one):
- Improvement

### Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):
#50727 new alias for function current_database and added new function current_schemas

### Documentation entry for user-facing changes

- [ ] Documentation is written (mandatory for new features)

> Information about CI checks: https://clickhouse.com/docs/en/development/continuous-integration/

### CODE REVIEW

File: src/Functions/currentSchemas.cpp [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+        return DataTypeArray(std::make_shared<DataTypeString>())
+                               .createColumnConst(input_rows_count, Array { db_name });
+    }
+};
+
+}
+
+REGISTER_FUNCTION(CurrentSchema)
+{
+    factory.registerFunction<FunctionCurrentSchemas>();
```
Comment (alexey-milovidov) [inline src/Functions/currentSchemas.cpp:64]: LGTM. But it also requires inline documentation.
See `asin.cpp` for example.

**Comment by robot-clickhouse** [comment]
This is an automated comment for commit 4a2f1ca45d1cf4f7f03774a07bd27b65996cae76 with description of existing statuses. It's updated for the latest CI running
The full report is available [here](https://s3.amazonaws.com/clickhouse-test-reports/51076/4a2f1ca45d1cf4f7f03774a07bd27b65996cae76/ci_running.html)
The overall status of the commit is 🟡 pending

<table><thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody><tr><td>CI running</td><td>A meta-check that indicates the running CI. Normally, it's in <b>success</b> or <b>pending</b> state. The failed status indicates some problems with the PR</td><td>🟡 pending</td></tr>
<tr><td>Mergeable Check</td><td>Checks if all other necessary checks are successful</td><td>🟢 success</td></tr>
</table>

**Comment by priera** [comment]
Hello @Eridanus117, 

I totally agree with your point. I just chose this issue because it seemed a good entry point for learning the code base. Maybe I shouldn't had open a PR, or at least it would had been polite to ask in advance. My apologies if that annoyed you. I have no problem on resigning of my PR in favor of yours.

**Comment by priera** [comment]
> > Hello @Eridanus117,
> > I totally agree with your point. I just chose this issue because it seemed a good entry point for learning the code base. Maybe I shouldn't had open a PR, or at least it would had been polite to ask in advance. My apologies if that annoyed you. I have no problem on resigning of my PR in favor of yours.
> 
> @priera Since we both have the common goal of improving clickhouse, I think it would be a better choice to let you continue this PR.
> 
> Have fun in clickhouse!

Perfect, thanks a lot! Sorry for the inconvenience, but this has been the first and last time ;)

**Comment by alexey-milovidov** [comment]
Ok, one small item remains:

Screenshot_20230619_034502

**Review by alexey-milovidov — APPROVED** [review]

**Comment by priera** [comment]
Hello,

I would like to know if this PR can be merged, Some tests failed but that seems related to the CI environment, and not  a direct cause of the changes introduced in this branch.

For consistency I think it would be better to declare the new function as currentSchemas, and then define an alias current_schemas. I will do a push with that change, so no need to open another PR for that.

**Comment by alexey-milovidov** [comment]
It is already approved but blocked by failing tests. It will be merged as soon as the tests are fixed.

**Comment by priera** [comment]
I updated the branch yesterday, and the test `00834_kill_mutation` failed. This is a related [output](https://s3.amazonaws.com/clickhouse-test-reports/51076/f61d2776eb719948d1ffe1b1447e4dd421ac8980/stateless_tests__ubsan__[2_2].html).

After checking, it looks like that recently this test has become flaky, so I created a [ticket](https://github.com/ClickHouse/ClickHouse/issues/51818) for that.

With this can this PR be merged?

**Review by alexey-milovidov — APPROVED** [review]
Perfect!

**Pull request merged**